dist/appForms-backbone.js

```diff
@@ -4430,12 +4431,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
+    if (displayIndex !== 0 || hasCustomPageName) {
```

Comment:
If there is no page title specified for the form, I don't think it should display the default "Page 1" title if there is only a single page.